histomicstk/cli/BackgroundIntensity/BackgroundIntensity.py

```diff
@@ -15,8 +19,25 @@ def main(args):
 
     utils.create_dask_client(args)
     I_0 = background_intensity(**kwargs)
-    with open(args.returnParameterFile, 'w') as f:
-        f.write('BackgroundIntensity = ' + ','.join(map(str, I_0)) + '\n')
+
+    #
+    # Write annotation file
+    #
+
+    print('\n>> Writing annotation file ...\n')
+
+    annotation = {
+        'name': 'BackgroundIntensity',
+        'intensity_values': list(map(str, I_0)),
```

Comment:
This needs to be in the `attributes` dictionary.  Additionally, rather than casting this to a string, we should probably just do `'intensity_values': I_0.tolist(),`.